caddyhttp/httpserver/plugin.go

```diff
@@ -42,6 +42,7 @@ func init() {
 	flag.DurationVar(&GracefulTimeout, "grace", 5*time.Second, "Maximum duration of graceful shutdown")
 	flag.BoolVar(&HTTP2, "http2", true, "Use HTTP/2")
 	flag.BoolVar(&QUIC, "quic", false, "Use experimental QUIC")
+	flag.BoolVar(&CaseSensitivePath, "case-sensitive-path", false, "Use cases to differentiate paths")
```

Comment:
Oops, tried to avoid using global variable, via pushing boolean `caseSensitivePath` into the context. Dropped this approach after I realized there's no guarantee init in [`middleware.go`](https://github.com/mholt/caddy/blob/master/caddyhttp/httpserver/middleware.go) will be called before the one in [`plugin.go`](https://github.com/mholt/caddy/blob/master/caddyhttp/httpserver/plugin.go). Left some code by mistake. Fixed.